trunk/examples/programs/regression/c/ACSL-ghostInitGlobal.c

```diff
@@ -0,0 +1,11 @@
+//#Safe
+// Author: schuessf@informatik.uni-freiburg.de
+// Date: 2023-09-18
+
+//@ ghost int i = 0;
+
+int main() {
+  int x = 0;
+  //@ assert x == i;
```

Comment:
Wie ist denn die Semantik für die Nutzung von Ghost-Variablen in Asserts? Beeinflusst man damit nicht das Programm? Sowohl in dem Sinn, dass ein failure auftreten kann; als auch in dem Sinn, dass wir ggf eine `assume x!=i`-Kante einfügen.

Reply:
We should have a look at the ACSL spec for this. Of course `//@ assert` influences the program, but in my intuition (from a witness perspective), ghost code should not. But it may well be that in this specific sense, ACSL allows such influence.

Reply:
But I mean here it is just the assert that influences the program and not the ghost statement, right? You are right, the ghost code should not influence any program variable or the control flow, but I cannot see, how this is possible.